pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -169,10 +169,10 @@ class RichProgressBarTheme:
     """
 
     text_color: str = "white"
-    progress_bar_complete: Union[str, Style] = "#6206E0"
-    progress_bar_finished: Union[str, Style] = "#6206E0"
+    progress_bar_completed_so_far: Union[str, Style] = "#6206E0"
+    progress_bar_on_completion: Union[str, Style] = "#6206E0"
```

Comment:
```suggestion
    progress_bar: Union[str, Style] = "#6206E0"
    progress_bar_finished: Union[str, Style] = "#6206E0"
```
These are my suggestions. `progress_bar` is indicative enough what it means. Adding `complete*` makes it more confusing. 

cc: @SeanNaren

Reply:
@Raahul-Singh I would appreciate if you could what these variables mean in the docstring!

Reply:
Sure!

Reply:
> These are my suggestions. `progress_bar` is indicative enough what it means. Adding `complete*` makes it more confusing.
> 
> cc: @SeanNaren

I have added these in the commit 71ea449

Reply:
I like it :)